### PR TITLE
[api] Test multiple versions

### PR DESCRIPTION
**Pull request by driusan**
This attempts to add tests for multiple versions of the API. The tests from v0.0.4-dev are copied to v0.0.3 classes and the version in the new classes changed to v0.0.3.

### CODE REVIEW

**Comment by driusan** [comment]
I don't think this should have passed. Needs investigation.

**Comment by driusan** [comment]
@kongtiaowang can you review this?

File: test/phpunit.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <phpunit
         bootstrap="bootstrap.php"
         verbose="true"
+        testdox="true"
```
Comment (kongtiaowang) [inline test/phpunit.xml:5]: What is its purpose here? It was not found to be used. The rest of the code I think is fine, except for this little problem.

**Review by kongtiaowang — APPROVED** [review]
LGTM

**Pull request merged**